src/router.js

```diff
@@ -21,6 +22,12 @@ import UserProfile from './views/UserProfile.vue';
 const routes = [
   {
     path: '/',
+    name: 'landing',
+    component: Landing,
+    meta: { title: 'Welcome', fullScreen: true, layoutClass: 'landing-page' },
```

Comment:
```suggestion
    meta: { fullScreen: true, layoutClass: 'landing-page' },
```
Define it using `metaInfo` instead:
https://github.com/aeternity/superhero-ui/blob/f70ec4f9bd3b916e97919ce1a6d620dffe1e8d14/src/views/FAQ.vue#L40-L42